packages/eslint/core.js

```diff
@@ -60,8 +60,8 @@ module.exports = {
     ],
 
     // who doesn't love a good nested ternary?
```

Comment:
you have to change the snarky comment, too 😉

Reply:
actually, I think the comment still makes sense since it's transitioned to warn, not error